std/utf.d

```diff
@@ -1905,6 +1908,14 @@ unittest
 }
 /* =================== Encode ======================= */
 
+private dchar _utfException(UseReplacementDchar useReplacementDchar)(string msg, dchar c)
```

Comment:
Maybe it's a good idea for some algorithms @WalterBright, but clearly it doesn't get rid of double validation as seens in the example.
When I first `readText` a file and then do `byDchar` the latter has no business to revalidate the string.
I made an ER https://issues.dlang.org/show_bug.cgi?id=14919.